thrift/matching.thrift

```diff
@@ -22,6 +22,12 @@ include "shared.thrift"
 
 namespace java com.uber.cadence.matching
 
+// TaskSource is the source from which a task was produced
+enum TaskSource {
+    HISTORY    // Task produced by history service
```

Comment:
thrift enum may need a following comma at the end of value, see https://diwakergupta.github.io/thrift-missing-guide/

Reply:
interesting. I think you are right - but it generated code just fine when I tested this without the comma.